# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/snap/fractional-filter.yaml
# Commit: fc2b3d6448bc
# Full Hash: fc2b3d6448bcc3575fffb1abc923e7e703aeef54
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-24 03:51:41
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Part 2 - Performance and quality fixes for part 1. r=gfx-reviewers,nical
#   
#   * Add support for local scale factors to a surface, allowing it to
#     be rasterized in root coordinate space. This allows snapping to
#     work across surfaces where the surface transform is a fractional
# ==============================================================================

diff -r 8c70bc92ad36 -r fc2b3d6448bc gfx/wr/wrench/reftests/snap/fractional-filter.yaml
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/wr/wrench/reftests/snap/fractional-filter.yaml	Wed Feb 23 20:49:27 2022 +0000
@@ -0,0 +1,14 @@
+# Verify that a fractional transform that is attached to an off-screen
+# surface is able to snap correctly
+root:
+  items:
+    - type: stacking-context
+      bounds: [0, 0, 1920, 1200]
+      transform: translate(-45, 7.491675)
+      items:
+      - type: stacking-context
+        filters: [opacity(0.9)]
+        items:
+          - text: "This should be pixel aligned!"
+            origin: 416.543499 160.008325
+            size: 16